### DISCUSSION TITLE
Converting parent_id to name in the table

### DISCUSSION THREAD

**Discussion by berkay-huz** [filamentphp/filament#8206] [General]
->columns([
                TextColumn::make('name')->limit('50')->sortable()->searchable(),
                TextColumn::make('parent_id'),
                TextColumn::make('slug')->limit('50')
            ]) 

In these codes, in the 'parent_id' section, it should display the 'name' equivalent to whichever 'name' corresponds, let it display the 'name'.

**Comment by leandrocfe** [comment]
https://filamentphp.com/docs/3.x/tables/columns/relationships#displaying-data-from-relationships

**Comment by alchemizt** [comment]
Heres how I did it, in the model, in my case Taxonomy is the model, add this:
`     public function parent()
     {
        return $this->belongsTo(Taxonomy::class, 'parent_id');
     }`

then in the resource, change it to this:
`Tables\Columns\TextColumn::make('parent.name')`


[https://medium.com/@miladev95/using-parent-child-recursive-relationships-in-laravel-2124b8885719](url)
